crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -647,6 +709,38 @@ where
     }
 }
 
+pub struct CloneBundle<T> {
+    pub source: Entity,
+    pub target: Entity,
+    pub _bundle: PhantomData<T>,
+}
+
+impl<T> Command for CloneBundle<T>
+where
+    T: Bundle + Clone + 'static,
+{
+    fn write(self, world: &mut World) {
+        let bundle = if let Some(mut source) = world.get_entity_mut(self.source) {
+            let todo: Option<&T> = todo!("we need a get_bundle on EntityMut");
```

Comment:
We're lacking a `EntityRef::get_bundle` here?

Reply:
Yep. If the API doesn't have it, just add it as part of this PR.